docs/product-manuals/zeebe/deployment-guide/operations/resource-planning.md

```diff
@@ -75,72 +73,73 @@ Environment Variables
 
 Other factors can be observed in a production-like system with representative throughput.
 
-If you want to know where to look, by default this data is stored in
+By default, this data is stored in the following:
 
-- `segments` - the data of the log split into segments. The log is only appended - its data can be deleted when it becomes part of a new snapshot.
-- `state` - the active state. Deployed processes, active process instances, etc. Completed process instances or jobs are removed.
-- `snapshot` - a state at a certain point in time
+- `segments` - The data of the log split into segments. The log is only appended, and its data can be deleted when it becomes part of a new snapshot.
+- `state` - The active state. Deployed processes, active process instances, etc. Completed process instances or jobs are removed.
+- `snapshot` - A state at a certain point in time.
 
 > **Pitfalls**
 >
-> If you want to avoid exploding your disk space usage, here are a few pitfalls to avoid:
+> To avoid exploding your disk space usage, here are a few pitfalls to avoid:
```

Comment:
I still find the word "exploding" strange. Does that make sense to you, as a native speaker?

Reply:
This is a little awkward. How's this:

"To avoid exceeding your disk space, here are a few pitfalls to avoid"